tools/crm_resource.c

```diff
@@ -396,12 +395,13 @@ static GOptionEntry query_entries[] = {
     { "locate", 'W', G_OPTION_FLAG_NO_ARG, G_OPTION_ARG_CALLBACK, flag_cb,
       "Show node(s) currently running resource",
       NULL },
-    { "stack", 'A', G_OPTION_FLAG_NO_ARG, G_OPTION_ARG_CALLBACK, flag_cb,
-      "Display the (co)location constraints that apply to a resource\n"
-      INDENT "and the resources is it colocated with",
-      NULL },
     { "constraints", 'a', G_OPTION_FLAG_NO_ARG, G_OPTION_ARG_CALLBACK, flag_cb,
-      "Display the (co)location constraints that apply to a resource",
+      "Display the location and colocation constraints that apply to a\n"
+      INDENT "resource, and if --recursive is specified, to the resources\n"
+      INDENT "directly or indirectly involved in those colocations",
+      NULL },
```

Comment:
This goes recursively in both directions, right?

Reply:
yep